src/scripts/main.js

```diff
@@ -1,3 +1,135 @@
 'use strict';
 
-// write code here
+// sort function
+const tHeadTags = document.querySelectorAll('th'); // Вибираємо всі заголовки
+const tbody = document.querySelector('tbody'); // Тіло таблиці
+const rows = tbody.querySelectorAll('tr'); // Всі рядки
+
+const sortMode = {}; // Об'єкт для відстеження стану сортування
+
+tHeadTags.forEach((tag, thIndex) => {
+  // Початковий стан сортування — за зростанням
+  sortMode[thIndex] = 'asc';
+
+  tag.addEventListener('click', () => {
+    const direction = sortMode[thIndex];
+
+    if (direction === 'asc') {
+      sortTable(thIndex, 'asc');
+      sortMode[thIndex] = 'desc'; // Змінюємо стан
+    } else {
+      sortTable(thIndex, 'desc');
+      sortMode[thIndex] = 'asc'; // Змінюємо стан
+    }
+  });
+});
+
+function sortTable(indx, mode) {
+  const table = document.querySelector('table');
+  const content = Array.from(table.rows).slice(1, -1); // Пропускаємо заголовки
+
+  content.sort((a, b) => {
+    const cellA = a.cells[indx].innerText.trim();
+    const cellB = b.cells[indx].innerText.trim();
+
+    if (mode === 'asc') {
+      return cellA > cellB ? 1 : -1;
+    } else {
+      return cellA < cellB ? 1 : -1;
+    }
+  });
+
+  // Оновлюємо порядок рядків у таблиці
+  content.forEach((row) => table.appendChild(row));
+}
+
+// ACTIVE FOR ROWS
+Array.from(rows).forEach((element, index) => {
+  element.addEventListener('click', () => {
+    Array.from(rows).forEach((row) => {
+      row.classList.remove('active');
+    });
+    element.classList.add('active');
+  });
+});
+
+// CHECK TEXT FROM FORM
+const form = document.querySelector('.new-employee-form');
+
+form.addEventListener('submit', function (e) {
+  e.preventDefault();
+
+  let isValid = true;
+  const errors = [];
+
+  const inputName = form.querySelector('input[data-qa="name"]').value;
+  const age = form.querySelector('input[data-qa="age"]').value;
+  const position = form.querySelector('input[data-qa="position"]').value;
+  const office = form.querySelector('input[data-qa="office"]').value;
+  const salary = form.querySelector('input[data-qa="salary"]').value;
```

Comment:
The `salary` input value is a string. Consider converting it to a number if you plan to perform any numerical operations or comparisons.